fe/fe-core/src/main/java/com/starrocks/common/util/Util.java

```diff
@@ -321,8 +322,10 @@ public static String getResultForUrl(String urlStr, String encodedAuthInfo, int
                                          int readTimeoutMs) {
         StringBuilder sb = new StringBuilder();
         InputStream stream = null;
+        String safeUrl = urlStr;
         try {
             URL url = new URL(urlStr);
+            safeUrl = WebUtils.sanitizeHttpReqUri(urlStr);
```

Comment:
should only do the url sanitize when necessary, no need to do it for very request.

Reply:
![image](https://github.com/user-attachments/assets/e7bbe015-c95f-4bc0-8d9b-429ec3513509)
This method is only used for this cluster_id token url。

Reply:
it is generally not a good idea to have assumptions to the caller's behavior. Although in this method, it may not that critical to create an object that may or may not used at all.

Reply:
> it is generally not a good idea to have assumptions to the caller's behavior. Although in this method, it may not that critical to create an object that may or may not used at all.

Do you have any good suggestions? Look forward to your advice